tools/scripts/language-server/generate-stdlib.mjs

```diff
@@ -0,0 +1,73 @@
+// SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+//@ts-check
```

Comment:
What is this?

Reply:
[Enables the typescript checking on js files :) 
](https://www.typescriptlang.org/docs/handbook/intro-to-js-ts.html)